src/Symfony/Component/Validator/Constraints/MacAddress.php

```diff
@@ -0,0 +1,55 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\Validator\Constraints;
+
+use Symfony\Component\Validator\Constraint;
+use Symfony\Component\Validator\Exception\InvalidArgumentException;
+
+/**
+ * Validates that a value is a valid MAC address.
+ *
+ * @Annotation
+ * @Target({"PROPERTY", "METHOD", "ANNOTATION"})
+ *
+ * @author Ninos Ego <me@ninosego.de>
+ */
+#[\Attribute(\Attribute::TARGET_PROPERTY | \Attribute::TARGET_METHOD | \Attribute::IS_REPEATABLE)]
+class MacAddress extends Constraint
+{
+    public const INVALID_MAC_ERROR = 'a183fbff-6968-43b4-82a2-cc5cf7150036';
+
+    protected const ERROR_NAMES = [
+        self::INVALID_MAC_ERROR => 'INVALID_MAC_ERROR',
+    ];
+
+    public string $message = 'This is not a valid MAC address.';
```

Comment:
this message should be added to the corresponding translation file I suppose

Reply:
How can I regenerate the translation files? :angel:

Reply:
you should add it to the `src/Symfony/Component/Validator/Resources/translations/validators.en.xlf` file on branch 5.4 (and to the translation files in the language you know at the same time)